devlib/target.py

```diff
@@ -934,6 +934,10 @@ class AndroidTarget(Target):
     os = 'android'
     ls_command = ''
 
+    # See https://developer.android.com/reference/android/content/Intent.html#setFlags(int)
```

Comment:
I put those here but I'm not sure it's the best place. Should that live in utilities/android ?

Reply:
Since this is a collection of Intent's flags... should not this belong to an "intent" module (meaning python module, not devlib)? Something that along time can be improved to be a proxy of the link you reported above.

Reply:
Those would definitely benefit from being properly separated. *utills/android.py* has some definitions of Android versions, but IMO those flags are more meant for internal use whereas it makes sense to expose Android versions. 

As for a dedicated intent module, I'm not sure if it's worth creating one just for them. @marcbonnici / @setrofim, your thoughts on where to store these nasties ?

Reply:
It would certainly be nice to have somewhere to contain these nicely however I'm inclined to agree that it might not be worth creating something external for just these couple of values unless it is likely that many more of these will be required in the future and in other locations?

I might suggest moving them to the beginning of the file with the other global variables and adding a prefix to identify them rather than have them as class attributes for now? Although I'm not sure if that would make things much cleaner.

Reply:
I think that, for now, `utils/android.py` is the right place for them, as that's where we stick the various bits of Android miscellanea. In the future, it might make sense to  reorganise things further. However, I agree with @valschneider  and @marcbonnici that it's probably not worth creating a separate module just for those (e.g. at the moment, it's not clear whether it would make more sense to have `utils/android/intent.py` or `utils/android/flags.py`).